### PR TITLE
Locales: Update Hungarian Translations hu_HU

### PR DESCRIPTION
**Pull request by dio-gh**
Complete rewrite of the Hungarian translation for PCSX2's 1.6 release.

### CODE REVIEW

**Comment by RedDevilus** [comment]
Nice, u can keep WIP for now but can u change title to:
Locales: Update Hungarian Translations hu_HU
I'll check it in a bit if the checks are good. 

You can always do squash to reduce commits ( pick and s in the pdf)

**Comment by dio-gh** [comment]
I can if needed, though I specifically tried to follow that naming scheme with the commit name in specific, since that's the one that will end up in the commit list, not the PR name.

**Comment by RedDevilus** [comment]
Well fair enough but its easier to see on github a bit and not for dev builds like u said for the commit log.

**Comment by RedDevilus** [comment]
Mo files should be in pcsx2\bin\Langs\hu_HU and not locales.

File: locales/hu_HU/pcsx2_Main.po [hunk 1]
```diff
@@ -8,15 +8,15 @@ msgstr ""
 "Project-Id-Version: PCSX2 0.9.8\n"
```
Comment (lightningterror) [inline locales/hu_HU/pcsx2_Main.po]: 1.6.0

Comment (dio-gh) [reply locales/hu_HU/pcsx2_Main.po]: Fixed.

**Comment by RedDevilus** [comment]
Seems good. I'll wait for the review for LT and possible merge.

**Comment by dio-gh** [comment]
I mean, the merge could wait until 1.6 is actually about to drop, cause when I'll have some time in the coming days I'd definitely like to add more stuff (hence why I marked it WIP).

**Comment by lightningterror** [comment]
I won't merge it until the last day comes for release then.

**Comment by dio-gh** [comment]
Finalized it, ready for merge.

File: locales/hu_HU/pcsx2_Iconized.po [hunk 2]
```diff
@@ -5,71 +5,70 @@
...
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2011-04-17 17:56+0100\n"
-"Last-Translator: Delirious <delirious@freemail.hu>\n"
-"Language-Team: Delirious <delirious@freemail.hu>\n"
-"Language: \n"
+"PO-Revision-Date: 2020-03-18 20:58+0100\n"
+"Last-Translator: dio\n"
+"Language-Team: \n"
+"Language: hu\n"
```
Comment (RedDevilus) [inline locales/hu_HU/pcsx2_Iconized.po]: should be hu_HU
```suggestion
"Language: hu_HU\n"
```

File: locales/hu_HU/pcsx2_Main.po [hunk 3]
```diff
@@ -5,18 +5,19 @@
...
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2011-04-17 16:35+0100\n"
-"Last-Translator: Delirious <delirious@freemail.hu>\n"
-"Language-Team: Delirious <delirious@freemail.hu>\n"
-"Language: \n"
+"PO-Revision-Date: 2020-03-18 21:24+0100\n"
+"Last-Translator: dio\n"
+"Language-Team: \n"
+"Language: hu\n"
```
Comment (RedDevilus) [inline locales/hu_HU/pcsx2_Main.po]: imo
```suggestion
"Language: hu_HU\n"
```

File: locales/hu_HU/pcsx2_Iconized.po (hunk 2 above)
Comment (dio-gh) [reply locales/hu_HU/pcsx2_Iconized.po]: Fixed. No idea why Poedit keeps two separate Hungarian locales, Hungarian is only spoken at Hungary in official capacity.

File: locales/hu_HU/pcsx2_Main.po (hunk 3 above)
Comment (dio-gh) [reply locales/hu_HU/pcsx2_Main.po]: Fixed.

Comment (dio-gh) [reply locales/hu_HU/pcsx2_Main.po]: Fixed.

**Pull request merged**